src/video_core/host1x/host1x.h

```diff
@@ -0,0 +1,58 @@
+// Copyright 2022 yuzu Emulator Project
+// Licensed under GPLv3 or any later version
+// Refer to the license.txt file included.
```

Comment:
Oh wow 2022s are becoming consistent :)

```suggestion
// SPDX-FileCopyrightText: 2022 yuzu Emulator Project
// SPDX-License-Identifier: GPL-3.0-or-later
```